cypress/e2e/ModelizerPage/TextView/file/open.file.feature

```diff
@@ -1,32 +1,23 @@
-Feature: Test file explorer and file tabs
+Feature: Test modelizer text view: open file/folder
```

Comment:
can't open a folder, only expand. BUT can't expand a file either. It's only possible to double click on both but the result won't be the same.
AND
Does not match file name.
AND
Folder text missing.